SwiftTweaks/FloatingTweakGroupViewController.swift

```diff
@@ -12,7 +12,7 @@ import UIKit
 // MARK: - FloatingTweaksWindowPresenter
 
 internal protocol FloatingTweaksWindowPresenter {
-	func presentFloatingTweaksUIForTweakGroup(tweakGroup: TweakGroup)
+	func presentFloatingTweaksUIForTweakGroup(_ tweakGroup: TweakGroup)
```

Comment:
The Swift 3 convention here would be `func presentFloatingTweaksUI(forTweakGroup tweakGroup: TweakGroup)`